src/irc/core/irc-nicklist.c

```diff
@@ -510,7 +601,10 @@ void irc_nicklist_init(void)
 {
 	signal_add_first("event nick", (SIGNAL_FUNC) event_nick);
 	signal_add_first("event 352", (SIGNAL_FUNC) event_who);
+	signal_add_first("event 354", (SIGNAL_FUNC) event_whox_743);
 	signal_add("silent event who", (SIGNAL_FUNC) event_who);
+	signal_add("silent event whox", (SIGNAL_FUNC) event_whox_743);
+	signal_add("silent event whox useraccount", (SIGNAL_FUNC) event_whox_745);
```

Comment:
What do 743 and 745 mean here?

Reply:
Good point! These are arbitrary 3-digit numerics that the client should choose for their WHOX queries. I threw dice to get them. Make them a constant? Choose different numbers?

Reply:
I thought about encoding the selected fields in the number but 1000 numbers is not enough for all combinations

Reply:
Hahah dice, amazing.

How about this: Put both WHOX commands in some header, something like this

```
/* identifiers chosen by fair dice roll, guaranteed to be random */
#define WHOX_FULL_ID "743"
#define WHOX_USERACCOUNT_ID "745"

/* some explanation here about WHOX because it's a bullshit spec */
#define WHOX_FULL_CMD "WHO %s %%tcuhnfdar," WHOX_FULL_ID
#define WHOX_USERACCOUNT_CMD "WHO %s %%tna," WHOX_USERACCOUNT_ID
```

And rename the callbacks from event_whox_745 to event_whox_useraccount, etc.

Reply:
thanks, I implemented your suggestion